libusb/os/windows_winusb.c

```diff
@@ -3067,8 +3067,41 @@ static void WINAPI winusbx_native_iso_transfer_continue_stream_callback(struct l
 	} else {
 		// If the transfer wasn't successful we reschedule the transfer while forcing it
 		// not to continue the stream. This might results in a 5-ms delay.
-		transfer_priv->iso_break_stream = TRUE;
-		libusb_submit_transfer(transfer);
+		libusb_transfer_cb_fn postSubmissionHandler = NULL;
+		bool resubmitTransfer= TRUE;
+
+		if (transfer->callback) {
+			// Notify the user callback that a resubmission is about to take place
+			// This gives it an option to interfer or schedule submission of other transfers after this one
+			transfer->status = LIBUSB_TRANSFER_STALL;
```

Comment:
Question: How could the user differentiate in between a real "Stall" (could happen) and this indication? Is there another value to look at for differentiating the 2 cases ?
EDIT: OK I can see from documentation that STALL is not currently an option for ISO transfer.